pkg/plugin/plugin.go

```diff
@@ -78,7 +78,7 @@ func (p *plugin) Validate(ctx context.Context, new, old *api.OpenShiftManagedClu
 		// don't call checkIfClusterWillRefresh() if we already have errors.
 		return
 	}
-	if !externalOnly && old != nil {
```

Comment:
this was preventing the validation below as these 2 conditions never happen

Reply:
this needs some discussion as in the real rp clients don't exist in the first validate.

Reply:
Where did we land with this?

Reply:
i am changing it to (pending confirmation from Jack)
```
if !externalOnly && new.Properties.ProvisioningState != api.Creating {
```